packages/0x.js/package.json

```diff
@@ -21,7 +21,8 @@
         "generate_contract_wrappers": "abi-gen --abis 'src/compact_artifacts/@(Exchange|Token|TokenTransferProxy|EtherToken|TokenRegistry|DummyToken).json' --template ../contract_templates/contract.handlebars --partials '../contract_templates/partials/**/*.handlebars' --output src/contract_wrappers/generated --backend ethers && prettier --write 'src/contract_wrappers/generated/**.ts'",
         "lint": "tslint --project .",
         "test:circleci": "run-s test:coverage",
-        "test": "run-s clean test:commonjs",
+        "test": "run-s test:commonjs",
```

Comment:
This __package.json__ file is a little different than the rest. Is it okay to skip the build step in `test:commonjs`?

Reply:
Yes